server/routes/status.ts

```diff
@@ -127,55 +130,54 @@ const status = async (
     include: [models.Address, models.OffchainAttachment],
   });
 
-  const visiblePrivateCommunityIds = Array.from(
-    roles.map((role) => role.offchain_community_id)
-  ); // both private and public
-  const privateCommunities = await models.OffchainCommunity.findAll({
-    where: {
-      id: {
-        [Op.in]: visiblePrivateCommunityIds,
-      },
-      privacy_enabled: true,
-    },
-    include: [
-      {
-        model: models.OffchainTopic,
-        as: 'topics',
-      },
-    ],
-  });
+  // const visiblePrivateCommunityIds = Array.from(
+  //   roles.map((role) => role.offchain_community_id)
+  // ); // both private and public
+  // const privateCommunities = await models.OffchainCommunity.findAll({
+  //   where: {
+  //     id: {
+  //       [Op.in]: visiblePrivateCommunityIds,
+  //     },
+  //     privacy_enabled: true,
+  //   },
+  //   include: [
+  //     {
+  //       model: models.OffchainTopic,
+  //       as: 'topics',
+  //     },
+  //   ],
+  // });
   const allCommunities: any = _.uniqBy(
-    publicCommunities.concat(privateCommunities),
+    publicCommunities,
+    // publicCommunities.concat(privateCommunities),
     'id'
   );
+  console.log('blah2')
 
   const threadCount = {};
   const threadCountQueryData: ThreadCountQueryData[] =
     await models.sequelize.query(
       `
-SELECT CONCAT("OffchainThreads".chain, "OffchainThreads".community), COUNT("OffchainThreads".id)
-  FROM "OffchainThreads"
-  LEFT JOIN "OffchainCommunities"
-    ON "OffchainThreads".community = "OffchainCommunities".id
-WHERE "OffchainThreads".updated_at > :thirtyDaysAgo
-  AND "OffchainThreads".deleted_at IS NULL
-  AND NOT "OffchainThreads".pinned
-  AND ("OffchainThreads".chain IS NOT NULL
-    OR NOT "OffchainCommunities"."privacy_enabled"
-    OR "OffchainCommunities".id IN(:visiblePrivateCommunityIds))
-GROUP BY CONCAT("OffchainThreads".chain, "OffchainThreads".community);
-`,
+      SELECT "OffchainThreads".chain, COUNT("OffchainThreads".id) 
```

Comment:
I have to wonder if we can sequelize these raw SQL queries throughought now that we don't have to deal with the dual chain/community, or whether they still need to be raw.